config.py

```diff
@@ -64,7 +64,7 @@ def main():
     yaml.safe_dump(config_data, open('./sysconfigs/config.yaml', 'w'), default_flow_style=False)
 
     # copy modified config.yaml file in site scope of spack instance
-    os.popen('cp -rf sysconfigs/config.yaml ' + args.idir + '/spack/etc/spack')
+    os.popen('cp -rf sysconfigs/' + args.machine.replace('admin-', '') + '/config.yaml ' + args.idir + '/spack/etc/spack')
```

Comment:
Might be a good opportunity to use `subprocess.run` instead :P

Reply:
What you mean?
...
o.k. o.k.